client/src/components/App/App.js

```diff
@@ -135,33 +139,32 @@ export default function App() {
               badges,
               questsByVillager,
               questsByAdventurer,
-              adventurers
+              adventurers,
             });
           }
         )
         .catch((err) => console.log(err));
     } else {
-      const getUserQuest = axios.get('/quests');
-      const getVillagers = axios.get('/villagers');
-      const getClasses = axios.get('/classes');
-      Promise.all([
-        getUserQuest,
-        getVillagers,
-        getClasses
-      ]).then(
-        ([
-          { data: userQuests },
-          { data: villagers },
-          { data: classesData }
-        ]) => setState((prevState) => {
-          return {
-            ...prevState,
-            userQuests,
-            villagers,
-            classesData
-          };
-        })
-      ).catch(e => console.log(e));
+      const getUserQuest = axios.get("/quests");
+      const getVillagers = axios.get("/villagers");
+      const getClasses = axios.get("/classes");
+      Promise.all([getUserQuest, getVillagers, getClasses])
+        .then(
+          ([
+            { data: userQuests },
+            { data: villagers },
+            { data: classesData },
+          ]) =>
+            setState((prevState) => {
+              return {
+                ...prevState,
+                userQuests,
+                villagers,
+                classesData,
+              };
+            })
+        )
+        .catch((e) => console.log(e));
```

Comment:
Simpler?

```suggestion
      const userQuest = await axios.get("/quests").data;
      const villagers = await axios.get("/villagers").data;
      const classesData = await axios.get("/classes").data;
      setState((prevState) => {
              return {
                ...prevState,
                userQuests,
                villagers,
                classesData,
              };
            });
```

Reply:
NIce!

Reply:
This isn't working actually. Not sure why...

Reply:
```
TypeError: Cannot read property 'length' of undefined
ClassTabs
src/components/ClassTabs/ClassTabs.js:25
  22 | });
  23 | 
  24 | return (
> 25 |   <Tabs className="classTabOutline rounded" defaultActiveKey="profile"
     | ^  26 |     id="uncontrolled-tab-example"
  27 |     activeKey={key}
  28 |     onSelect={(k) => setKey(k)}>
```